public_dropin_environments/r_lang/requirements.txt

```diff
@@ -1,5 +1,5 @@
-numpy>=1.20.3
+numpy>=1.20.3,<2.0
 pandas<=2.0.3
 rpy2==3.5.8
 pyarrow>=0.14.1,<=14.0.1
-datarobot-drum[R]==1.15.0
+datarobot-drum[R]
```

Comment:
why removing this?

Reply:
I'll revert it, During my development I needed it, before I fixed the related harness to install DRUM from source code.